test/release/examples/primers/chplvis/prog1.chpl

```diff
@@ -0,0 +1,10 @@
+//  Example 1 using visual debug
```

Comment:
This file definitely should have more comments.  Being numerically first, it is probably what will be looked at first.